app/routes/schemas/sbi.js

```diff
@@ -5,6 +5,9 @@ module.exports = joi.object({
 }).error(errors => {
   errors.forEach(err => {
     switch (err.code) {
+      case 'number.greater':
+        err.message = 'The SBI is too short.'
+        break
       case 'number.less':
         err.message = 'The SBI is too long.'
```

Comment:
Suggest maybe changing the `less(999999999)` to be `less(1000000000)`.   As if I enter 999999999 it's not too long as it's the same amount of characters as a valid SBI.